.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: 🪲 Bug Report
+about: Report an issue to help us improve
+title: ""
+labels: "Issue-Bug"
+assignees: ""
+---
+
+### Description
+
+<!-- A brief description with a link to the page on the site where you found the issue. -->
+
+### Current Behavior
+
+<!-- A brief description of the current behavior of the issue. -->
+
+### Expected Behavior
+
+<!-- A brief description of what you expected to happen. -->
+
+### Screenshots
+
+<!-- Add screenshots, if applicable, to help explain your problem. -->
+
+### Environment:
+
+-   Operating System: [e.g., Windows, macOS, Linux]
+-   Browser (if applicable): [e.g., Chrome, Firefox, Safari]
+
+### Checklist
+
+-   [ ] I have read and followed the project's code of conduct.
+-   [ ] I have searched for similar issues before creating this one.
+-   [ ] I have provided steps to reproduce the issue.
+-   [ ] I have provided all the necessary information to understand and reproduce the issue.
+-   [ ] I am willing to contribute to the resolution of this issue if approved.
+
+---
```

Comment:
Love that last part (lines 38-42) 😄 However, I would put it in HTML comments, it doesn't have to appear in the body of the issue.